File: app/src/main/java/com/baro/AccountActivity.java
package com.baro;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.widget.GridView;
import android.widget.ImageButton;

public class AccountActivity extends AppCompatActivity {

    private ImageButton accountButton;
    private ImageButton followersButton;
    private ImageButton settingsButton;
    private ImageButton deleteLocalButton;
    private ImageButton deleteRemoteButton;
    private RecyclerView courseRecycleView;



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_account);

        configureAccountButton();
        configureFollowersButton();
        configureSettingsButton();
        configureDeleteLocalButton();
        configureDeleteRemoteButton();
        configureRecycleView();
    }

    private void configureAccountButton() {
        accountButton = findViewById(R.id.btn_account);

        accountButton.setOnClickListener(v -> {
            // TODO complete this method
            // Should allow user to change their picture
        });
    }


    private void configureFollowersButton() {
        followersButton = findViewById(R.id.btn_followers);

        followersButton.setOnClickListener(v -> {
            // TODO complete this method
            // Should allow user to see their followers
        });
    }


    private void configureSettingsButton() {
        settingsButton = findViewById(R.id.btn_settings);

        settingsButton.setOnClickListener(v -> {
            // TODO complete this method
            // Should allow user to enter the settings page
        });
    }


    private void configureDeleteLocalButton() {
        deleteLocalButton = findViewById(R.id.btn_delete_local);

        deleteLocalButton.setOnClickListener(v -> {
            // TODO complete this method
            // Should allow user to delete course from local
        });
    }

    private void configureDeleteRemoteButton() {
        deleteRemoteButton = findViewById(R.id.btn_delete_remote);

        deleteRemoteButton.setOnClickListener(v -> {
            // TODO complete this method
            // Should allow user to delete course from remote
        });
    }

    private void configure() {
        accountButton = findViewById(R.id.btn_account);

        accountButton.setOnClickListener(v -> {
            // TODO complete this method
            // Should allow user to change their picture
        });
    }


    private void configureRecycleView() {
        // TODO Implement this method
    }

}

File: app/src/main/java/com/baro/AccountActivity.kt
package com.baro

import android.os.Bundle
import android.view.View
import android.widget.ImageButton
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.RecyclerView


class AccountActivity : AppCompatActivity() {
    private lateinit var accountButton: ImageButton
    private lateinit var followersButton: ImageButton
    private lateinit var settingsButton: ImageButton
    private lateinit var deleteLocalButton: ImageButton
    private lateinit var deleteRemoteButton: ImageButton
//    private lateinit val courseRecycleView: RecyclerView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_account)
        configureAccountButton()
        configureFollowersButton()
        configureSettingsButton()
        configureDeleteLocalButton()
        configureDeleteRemoteButton()
        configureRecycleView()
    }

    private fun configureAccountButton() {
        accountButton = findViewById(R.id.btn_account)
        accountButton.setOnClickListener({ })
    }

    private fun configureFollowersButton() {
        followersButton = findViewById(R.id.btn_followers)
        followersButton.setOnClickListener(View.OnClickListener { v: View? -> })
    }

    private fun configureSettingsButton() {
        settingsButton = findViewById<ImageButton?>(R.id.btn_settings)
        settingsButton.setOnClickListener(View.OnClickListener { v: View? -> })
    }

    private fun configureDeleteLocalButton() {
        deleteLocalButton = findViewById<ImageButton?>(R.id.btn_delete_local)
        deleteLocalButton.setOnClickListener(View.OnClickListener { v: View? -> })
    }

    private fun configureDeleteRemoteButton() {
        deleteRemoteButton = findViewById<ImageButton?>(R.id.btn_delete_remote)
        deleteRemoteButton.setOnClickListener(View.OnClickListener { v: View? -> })
    }

    private fun configure() {
        accountButton = findViewById<ImageButton?>(R.id.btn_account)
        accountButton.setOnClickListener(View.OnClickListener { v: View? -> })
    }

    private fun configureRecycleView() {
        // TODO Implement this method
    }
}